html/cross-origin-embedder-policy/none-sw-from-none.https.html

```diff
@@ -0,0 +1,82 @@
+<!doctype html>
+<html>
+<script src=/resources/testharness.js></script>
+<script src=/resources/testharnessreport.js></script>
+<script src="/common/get-host-info.sub.js"></script>
+<script src="/service-workers/service-worker/resources/test-helpers.sub.js"></script>
+<script>
+const SCOPE = new URL(location.href).pathname;
+const SCRIPT =
+  'resources/sw.js?' +
+  `pipe=header(service-worker-allowed,${SCOPE})`;
+
+function remote(path) {
+  const REMOTE_ORIGIN = get_host_info().HTTPS_REMOTE_ORIGIN;
+  return new URL(path, REMOTE_ORIGIN + '/html/cross-origin-embedder-policy/');
+}
+
+promise_test(async (t) => {
+  const reg = await service_worker_unregister_and_register(t, SCRIPT, SCOPE);
+  const sw =
+    reg.installing ? reg.installing : reg.waiting ? reg.waiting : reg.active;
```

Comment:
|reg.installing| should always be valid at this point.

Reply:
Fixed.